imgoing-frontend/src/components/HomeMain/PlanItem.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import styled from 'styled-components/native';
+import CalloutTypo from '../typography/CalloutTypo';
+import CaptionTypo from '../typography/CaptionTypo';
+import ContentTypo from '../typography/ContentTypo';
+import SubheadlineTypo from '../typography/SubheadlineTypo';
+import DeleteModal from './DeleteModal';
+import MenuModal from './MenuModal';
+
+interface PlanItemProps {
+  data: {};
+}
+
+const PlanItem = (props: PlanItemProps) => {
+  const [isExpanded, setIsExpanded] = useState<boolean>(false);
+  const [isModalVisible, setIsModalVisible] = useState<boolean>(false);
+  const [isDeleteModalVisible, setIsDeleteModalVisible] = useState<boolean>(false);
+
+  return (
+    <PlanItemView
+      onPress={() => {
+        setIsExpanded(!isExpanded);
+      }}>
+      {/* 추후 컴포넌트화 고려 */}
+      <ExpandableBar>
+        {/* 시간에 따라 변경 */}
+        <TimeTag>
+          <CaptionTypo bold en color={'white'}>
+            PM
+          </CaptionTypo>
+        </TimeTag>
+        <SubheadlineTypo bold en color={'black'}>
+          11:12
+        </SubheadlineTypo>
+        {/* 고정여부에 따라 변경 */}
+        <ImageView source={require('../../../assets/images/pin.png')} />
+        {/* 현재 isExpanded === false 일 때, 모달 안열리는 문제 존재 */}
+        <OpenMenuButton onPress={() => setIsModalVisible(!isModalVisible)}>
```

Comment:
```suggestion
        <OpenMenuButton onPress={() => setIsModalVisible(true)}>
```

한가지 동작만 하는 경우에는 값으로 전달해주는 게 좋을 것같습니다!